packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -64,6 +64,23 @@ MeetingsUtil.checkForCorrelationId = (deviceUrl, locus) => {
   return false;
 };
 
+MeetingsUtil.parsePreferedWebexSite = (userPreferences) => {
+  let webexSite = null;
+
+  userPreferences.find((site) => {
+    const preferedSite = site.match(/"preferredWebExSite":"(\S+)"/);
```

Comment:
It's a shame that we have to be doing this kind of parsing. Why are they not returning proper json objects in the array? Should we maybe allow for whitespaces around the ":" character and also use the beginning of the string anchor, so it would be like this:
`match(/^"preferredWebExSite"\s*:\s*"(\S+)"/)`

Reply:
i checked with the native client and thats how it is currently which i really hate , No other option

changed the regex